src/redux/books/books.js

```diff
@@ -0,0 +1,76 @@
+import { v4 as uuidv4 } from 'uuid';
+
+// Initial book state
+const bookState = [
+  {
+    id: 1,
+    title: 'Toilet Paper',
+    category: 'Science',
+    author: 'Frank dejoin',
+    status: true,
+  },
+  {
+    id: 2,
+    title: 'Melborn Syndy',
+    category: 'Clothing',
+    author: 'Esther Adeleke',
+    status: false,
+  },
+  {
+    id: 3,
+    title: 'Metal Chair',
+    category: 'Adame',
+    author: 'Sule Marke',
+    status: false,
+  },
+
+  {
+    id: 4,
+    title: 'Daniel Ufeli',
+    category: 'Agent47',
+    author: 'Onos Mark',
+    status: true,
+  },
+];
```

Comment:
- [ ] kindly be noticed that you should set an initial state with an empty array.